libs/perun/components/src/lib/attributes-list/attribute-value/attribute-value-list/attribute-value-list.component.html

```diff
@@ -1,49 +1,53 @@
+  #chipList
+  cdkDropList
+  cdkDropListOrientation="vertical"
+  [cdkDropListDisabled]="dragDisabled"
+  (cdkDropListDropped)="drop($event); _sendEventToParent()">
+  <div class="d-flex flex-column">
     <mat-chip-row
       class="chip-word-break"
       *ngFor="let chip of values.slice(0, itemsShown)"
-      [selectable]="selectable"
       cdkDrag
       [cdkDragData]="chip"
       [removable]="removable"
       (removed)="remove(chip); _sendEventToParent()">
-      <mat-icon *ngIf="removable" class="handle-icon" (mousedown)="dragDisabled = false;"
-        >drag_indicator</mat-icon
-      >
       <div
         (click)="readonly && attribute.displayName !== undefined  && showValue(chip, attribute.displayName)"
         [class.cursor-pointer]="readonly && attribute.displayName !== undefined"
-        class="chip-text overflow-ellipsis">
-        {{chip}}
+        class="chip-body">
+        <mat-icon *ngIf="removable" class="handle-icon" (mousedown)="dragDisabled = false;"
+          >drag_indicator</mat-icon
+        >
+        <span class="chip-text">{{chip}}</span>
       </div>
-      <mat-icon *ngIf="removable" (click)="edit(chip)" class="edit-delete-icon ms-2">edit</mat-icon>
-      <mat-icon *ngIf="removable" (click)="remove(chip)" class="edit-delete-icon">cancel</mat-icon>
+      <button matChipTrailingIcon>
+        <mat-icon *ngIf="removable" (click)="edit(chip)" class="edit-delete-icon ms-2"
+          >edit</mat-icon
+        >
+      </button>
+      <button *ngIf="removable" matChipRemove>
+        <mat-icon class="edit-delete-icon">cancel</mat-icon>
+      </button>
     </mat-chip-row>
-    <div class="w-100" *ngIf="values.length > defaultItemsShown">
-      <span [matTooltip]="showMore ? 'Show less' : 'Show more'" [matTooltipPosition]="'above'">
-        <button class="ms-auto" mat-icon-button (click)="onShowChange()">
-          <mat-icon *ngIf="!showMore">more_horiz</mat-icon>
-          <mat-icon *ngIf="showMore">keyboard_arrow_up</mat-icon>
-        </button>
-      </span>
-    </div>
-    <mat-form-field *ngIf="!readonly" subscriptSizing="dynamic" class="w-100">
-      <input
-        matInput
-        class="w-100 m-0"
-        placeholder="New value"
-        [matChipInputFor]="chipList"
-        [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
-        [matChipInputAddOnBlur]="addOnBlur"
-        (matChipInputTokenEnd)="add($event)"
-        (change)="_sendEventToParent()" />
-    </mat-form-field>
-  </mat-chip-grid>
-</div>
+  </div>
+  <div class="w-100" *ngIf="values.length > defaultItemsShown">
+    <span [matTooltip]="showMore ? 'Show less' : 'Show more'" [matTooltipPosition]="'above'">
+      <button class="ms-auto" mat-icon-button (click)="onShowChange()">
+        <mat-icon *ngIf="!showMore">more_horiz</mat-icon>
+        <mat-icon *ngIf="showMore">keyboard_arrow_up</mat-icon>
+      </button>
+    </span>
+  </div>
+  <mat-form-field *ngIf="!readonly" subscriptSizing="dynamic" class="w-100">
```

Comment:
On this page is an error when the attribute is read-only due to required `matChipInputFor` for `mat-chip-grid`.
![image](https://user-images.githubusercontent.com/70371638/217239094-6cadb568-5740-49ad-97ab-0b860b574dfd.png)
You can just replace `*ngIf` by `[hidden]` and it should work correctly.
```suggestion
  <mat-form-field [hidden]="readonly" subscriptSizing="dynamic" class="w-100">
```